packages/hooks/use-popup-manager/index.ts

```diff
@@ -0,0 +1,30 @@
+import { ref, getCurrentInstance } from 'vue'
+import { useGlobalConfig } from '../use-global-config'
+
+let zIndex = 0
+export const usePopupManager = () => {
+  let globalInitialZIndex = 2000 // TODO: move to @element-plus/constants
```

Comment:
If you really want the `setGlobalInitialZIndex` to work, I think you should move this variable out of this hook body.

```ts
let zIndex = 0
let globalInitialZIndex = 2000

export const setGlobalInitialZIndex = (val) => {
  globalInitialZIndex = value
}
export const usePopupManager = () => {
  // Other contents
}
```

Reply:
Fixed.